release/src/module_dependencies/update_version_data.py

```diff
@@ -256,8 +256,8 @@ def update_stdlib_dashboard(module_details_json):
         "[![GitHub Release](" + GITHUB_BADGE_URL + "release/" + BALLERINA_ORG_NAME + "/" + module['name'] + ".svg?label=)]" + 
         "(" + BALLERINA_ORG_URL + module['name'] + "/releases)| " + 
 
-        "[![Build](" + BALLERINA_ORG_URL + module['name'] + "/workflows/Build/badge.svg)]" + 
-        "(" + BALLERINA_ORG_URL + module['name'] + "/actions?query=workflow%3ABuild)| " + 
+        "[![Timestamped Build](" + BALLERINA_ORG_URL + module['name'] + "/actions/workflows/build-timestamped-master.yml/badge.svg)]" + 
```

Comment:
Shall we use just `Build` for this? `Timestamped Build` is long and will shrink the table columns.

Reply:
Unfortunately, we cant edit this. It's based on the GitHub action name. However, this is temporary these builds will be changed to just "Build" after the alpha5 release